static/app/views/partials/header.html

```diff
@@ -20,8 +23,8 @@
                     <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
                     aria-expanded="false" id="org-dropdown">{{ (activeOrg | orgNameFilter) || "Organizations" }}&nbsp;<span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li ng-repeat="org in orgs" ng-click="showOrg(org)">
-                            <a class="org-name">{{org | orgNameFilter}}</a>
+                        <li ng-repeat="org in $parent.orgs">
```

Comment:
I think I need it because it is an ng-include.... It is a bit of a hack, we can improve it.